common/trinary/trit_byte.c

```diff
@@ -37,25 +37,22 @@ byte_t trits_to_byte(trit_t const *const trits, byte_t const cum,
   return trits_to_byte(trits, byte, num_trits - 1);
 }
 
-void _byte_to_trits(byte_t byte, trit_t *const trits, size_t const j,
-                    size_t const i) {
-  trit_t trit;
-  if (byte > (byte_radix[j] >> 1)) {
-    byte -= byte_radix[j];
-    trit = 1;
-  } else if (byte < -(byte_radix[j] >> 1)) {
-    byte += byte_radix[j];
-    trit = -1;
-  } else {
-    trit = 0;
+void _byte_to_trits(byte_t byte, trit_t *const trits, size_t j, size_t i) {
+  while (j < -1) {
```

Comment:
nice, but hacky, can you just >= 0 ?

Reply:
No because j is a size_t and therefore is always >= 0. -1 automatically takes the highest value, which will never be the requested size